specification/trace/semantic_conventions/messaging.md

```diff
@@ -220,7 +220,11 @@ If an intermediary broker is present, `service.name` and `peer.service` will not
 
 #### CloudEvents
 
-For [CloudEvents](https://cloudevents.io/) clients, the following attributes are defined:
+For [CloudEvents](https://cloudevents.io/) clients, the `messaging.system` MUST be set to `cloudevents`. The `messaging.destination` SHOULD be set to `<event type> <operation name>`. If known, the `messaging.protocol` MAY be set to the underlying protocol used by the CloudEvents client. 
```

Comment:
Can you suggest any values for the protocol?

Reply:
Not really, since it's not an enum anywhere (neither here in the semantic conventions, nor in the cloud events spec).